javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/CompilationUnitContext.java

```diff
@@ -128,6 +130,24 @@ public CompilationUnitContext(CompilationUnit wrappedNode, TypeSolver typeSolver
                     }
                 }
             }
+            // look for member classes/interfaces of types in this compilation unit
```

Comment:
Ok, this part is not super-obvious. Could you add a comment to this and the previous part to explain what were we looking for before and what are we looking for here?

Reply:
Sure.